docker/goreleaser/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM goreleaser/goreleaser
+
+COPY . /m3
```

Comment:
why not mount the directory instead?

Reply:
I think this was a hack to get around trying to mount a volume on root, I think I know how to fix it now though